### PR TITLE
fix: language row horizontal alignment

### PR DESCRIPTION
**Pull request by GeronimoJohn**
# Description

### Issue

On desktop it becomes a little bit hard to read the languages as some of them are very long. The long title gets fitted in small space resulting in a very hard to read text

### Solution

We are to render all languages in one on desktop, and on hover it should show a tooltip showing the full length of the title

### CODE REVIEW

**Review by scottkelso — APPROVED** [review]
Just left one code improvement.

File: libs/journeys/ui/src/components/SearchBar/SearchDropdown/RefinementGroups/RefinementGroup/RefinementGroup.tsx [hunk 1]
```diff
@@ -96,3 +107,46 @@ export function RefinementGroup({
...
+  useEffect(() => {
+    function checkTruncation(): void {
+      if (textRef.current != null) {
+        setIsTextTruncated(
+          textRef.current.scrollWidth > textRef.current.clientWidth
+        )
+      }
+    }
+
+    checkTruncation()
```
[the remark this answers is not in the document]
Comment (GeronimoJohn) [reply libs/journeys/ui/src/components/SearchBar/SearchDropdown/RefinementGroups/RefinementGroup/RefinementGroup.tsx:118]: Initially had it like that, though scrollWidth and clientWidth are not available during server-side rendering. Only becomes accessible after the component has mounted

[the remark this answers is not in the document]
Comment (scottkelso) [reply libs/journeys/ui/src/components/SearchBar/SearchDropdown/RefinementGroups/RefinementGroup/RefinementGroup.tsx:118]: In that case the function wrapper isn't adding any value here. 
```suggestion
  useEffect(() => {
    if (textRef.current != null) {
      setIsTextTruncated(
        textRef.current.scrollWidth > textRef.current.clientWidth
      )
    }
```

[the remark this answers is not in the document]
Comment (GeronimoJohn) [reply libs/journeys/ui/src/components/SearchBar/SearchDropdown/RefinementGroups/RefinementGroup/RefinementGroup.tsx:118]: We have to call the function when the window resizes

**Pull request merged**